cfme/infrastructure/virtual_machines.py

```diff
@@ -1666,10 +1694,27 @@ def step(self, *args, **kwargs):
 
 @navigator.register(InfraTemplateCollection, 'TemplatesOnly')
 class TemplatesAll(CFMENavigateStep):
-    VIEW = TemplatesOnlyAllView
     prerequisite = NavigateToSibling('All')
 
+    @property
+    def VIEW(self):  # noqa
+        from cfme.infrastructure.provider import ProviderTemplatesView
+        from cfme.infrastructure.provider import InfraProvider
```

Comment:
Can these be moved to the top of the file?

Reply:
I moved ProviderTemplatesView to top of file, but I'll get import errors if I try to move InfraProvider. I am wondering if this is something I should look into with another PR or if I am missing a simple fix that could be added here.